### DISCUSSION TITLE
Write an Internet RFC

### DISCUSSION THREAD

**Discussion by silverwind** [bluesky-social/atproto#1145] [Protocol (atproto)]
Every reputable Internet protocol should have an [RFC](https://en.wikipedia.org/wiki/Request_for_Comments). Are there plans for one for the AT protocol?

**Comment by bnewbold** [comment]
We do want to ultimately go through a formal standards body. This could be the IETF RFC process, or another body, like W3C. We do want to finish our reference implementation before we start that process though, which includes working federation with our production Bluesky service.
